client/src/main/scala/com.gu.contentapi.client/model/HttpResponse.scala

```diff
@@ -0,0 +1,14 @@
+package com.gu.contentapi.client.model
+
+import scala.concurrent.Future
+
+case class HttpResponse(body: Array[Byte], statusCode: Int, statusMessage: String)
+
+object HttpResponse {
+  val isValid = Set(200, 302) 
```

Comment:
I think that we should rename this to something that indicates success or acceptance because all HTTP status codes are "valid" in a response.